src/display.rs

```diff
@@ -11,16 +11,43 @@
 
 use libc;
 use base::CGError;
+use geometry::{CGRect, CGPoint, CGSize};
 
 pub type CGDirectDisplayID = libc::uint32_t;
 
 #[link(name = "ApplicationServices", kind = "framework")]
 extern {
     pub fn CGMainDisplayID() -> CGDirectDisplayID;
+    pub fn CGDisplayIsActive(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsAlwaysInMirrorSet(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsAsleep(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsBuiltin(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsInHWMirrorSet(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsInMirrorSet(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsMain(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsOnline(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsStereo(display: CGDirectDisplayID) -> bool;
```

Comment:
To my reading of the header file, these all return `boolean_t`; I'm having trouble figuring out exactly what that'd typedef'd to, but it seems to indicate it's just `int` on x86. Is `bool` the correct return type here?